src/app/game/tic-tac-toe/multi-player/page.js

```
@@ -1,254 +1,359 @@
+    }
+  };
 
+  // UI
     return (
+    <div className={styles.container}>
+      <h1 className={styles.heading}>Tic Tac Toe <span className={styles.multiplayerLabel}>(Multiplayer)</span></h1>
+
+      {/* LOBBY: Only show if no game is active */}
+      {!gameNumber && (
+        <div className={styles.lobby}>
+          <button className={styles.createBtn} onClick={handleCreateGame} disabled={loading}>Create Game</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)